flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolTest.java

```diff
@@ -587,7 +587,7 @@ public void testCheckIdleSlot() throws Exception {
 			assertThat(slotPool.offerSlot(taskManagerLocation, taskManagerGateway, slotToNotExpire),
 				Matchers.is(true));
 
-			clock.advanceTime(1L, TimeUnit.MILLISECONDS);
+			clock.advanceTime(2L, TimeUnit.MILLISECONDS);
```

Comment:
Idle condition is `currentRelativeTimeMillis - slotAndTimestamp.timestamp > idleSlotTimeout.toMilliseconds()`, thus clock should advance `timeout.toMilliseconds() + 1` to trigger idle.